data/run.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+#
+# Do nothing if sync destination is not defined
+#
+if [ -z "$SYNC_DESTINATION" ]; then
+    exit 0
```

Comment:
Am I right in thinking we don't need to tell the user that this has happened because it's a valid way to use the image, if you want?